docker/run_update_data.sh

```diff
@@ -8,14 +8,22 @@ set -eo pipefail
 
 # Fetch and update release information for these products (comma-delimited)
 PRODUCTS="firefox"
+CRONTABBER="docker-compose run crontabber python socorro/cron/crontabber_app.py"
+
 
 # Fetch release data
-docker-compose run crontabber python socorro/cron/crontabber_app.py --job=ftpscraper \
+${CRONTABBER} --job=ftpscraper \
        --crontabber.class-FTPScraperCronApp.products=${PRODUCTS}
 
 # Update featured versions data based on release data
-docker-compose run crontabber python socorro/cron/crontabber_app.py --job=featured-versions-automatic \
+${CRONTABBER} --job=featured-versions-automatic \
        --crontabber.class-FeaturedVersionsAutomaticCronApp.products=${PRODUCTS}
 
 # Fetch normalization data for versions we know about
 docker-compose run processor python docker/fetch_normalization_data.py
+
+# Truncate tables
+${CRONTABBER} --job=truncate-partitions
+
+# Create new weekly tables
+${CRONTABBER} --job=weekly-reports-partitions
```

Comment:
I'll add a comment, but I'm not sure that we can get rid of this if we ditch Postgres as a crashstorage. We have a bunch of tables that are in the `xyz_yyyymmdd` format and I think crashstorage only covers some of them.

So it's possible, but I think someone should verify that before we go remove things.